descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Real Time Availability Check Module",
+  "provides": [
+    {
+      "id": "${artifactId}",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/records",
+          "permissionsRequired": ["oai-pmh.records.collection.get"],
+          "modulePermissions": [
+            "inventory-storage.instances.collection.get",
+            "inventory-storage.instances.item.get",
+            "inventory-storage.items.collection.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/records/{id}",
+          "permissionsRequired": ["oai-pmh.records.record.get"],
+          "modulePermissions": [
+            "inventory-storage.instances.item.get",
+            "inventory-storage.items.item.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/repository_info",
+          "permissionsRequired": ["oai-pmh.identify.get"],
+          "modulePermissions": []
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oia/identifiers",
+          "permissionsRequired": ["oai-pmh.identifiers.collection.get"],
+          "modulePermissions": [
+            "inventory-storage.instances.collection.get",
+            "inventory-storage.instances.item.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/metadata_formats",
+          "permissionsRequired": ["oai-pmh.metadata-formats.collection.get"],
+          "modulePermissions": [
+            "inventory-storage.instances.collection.get",
+            "inventory-storage.instances.item.get",
+            "inventory-storage.items.collection.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/sets",
+          "permissionsRequired": ["oai-pmh.set.collection.get"],
+          "modulePermissions": []
+        }
+      ]
+    }
+  ],
+  "requires": [
+    {
+      "id": "item-storage",
+      "version": "6.0"
+    },
+    {
+      "id": "instance-storage",
+      "version": "5.0"
+    }
+  ],
+  "permissionSets": [
+    {
+      "permissionName": "oai-pmh.records.record.get",
```

Comment:
consider making this `oai-pmh.records.item.get`?  Not a big deal, but it might follow the naming convention a little closer.

Reply:
@craigmcnally,
it looks better.
Thanks!